test/e2e/tests/test_event_generator/test_run_shell_untrusted.py

```diff
@@ -16,6 +16,9 @@
     sinsp.generate_id(sinsp_example) for sinsp_example in sinsp_examples
 ]
 
+# modern probe doesn't support EXE_WRITABLE flag yet
```

Comment:
@LucaGuerra is this in your queue for Falco 0.35?

Reply:
yep, I've in on the roadmap, in Falco 0.35 the modern probe should reach feature parity :tada: